web/app.go

```diff
@@ -70,7 +70,8 @@ func NewAppWithDeps(host string, port int, deps Dependencies) (*App, error) {
 	engine.GET("/hosts/:name/ha-checks", NewHAChecksHandler(deps.consul))
 	engine.GET("/catalog", NewChecksCatalogHandler(deps.checksService))
 	engine.GET("/clusters", NewClusterListHandler(deps.consul))
-	engine.GET("/clusters/:id", NewClusterHandler(deps.consul))
+	engine.GET("/clusters/:id", NewClusterHandler(deps.consul, deps.checksService))
+	engine.POST("/clusters/:id", NewPostClusterHandler(deps.consul))
```

Comment:
maybe i would change the path to `/clusters/:id/checks`

Reply:
Yes, better. We might have other posts most probably for this cluster